snapfaas/src/fs/mod.rs

```diff
@@ -1439,6 +1545,32 @@ pub mod utils {
         }
     }
 
+    pub fn create_service<S: Clone + BackingStore, P: Into<self::path::Path>>(
+        fs: &FS<S>, base_dir: P, name: String, policy: Buckle, service_info: ServiceInfo) -> Result<(), Error> {
```

Comment:
I think it is better for this high-level call to take in ServiceInfo's member fields instead of a ServiceInfo instance.